Dictionaries/hrv_OCRFixReplaceList.xml

```diff
@@ -1501,9 +1502,9 @@
     <RegEx find="oktobr([au]|om|ima)" replaceWith="listopad$1" />
     <RegEx find="decembr([au]|ima)" replaceWith="prosinc$1" />
     <!-- pridjevi optimalan, minimalan i maksimalan nemaju komparativ i superlativ. -->
-    <RegEx find="([nN])aj(optimaln)ij([aeiou])" replaceWith="$2$3" />
-    <RegEx find="([nN])aj(minimaln)ij([aeiou])" replaceWith="$2$3" />
-    <RegEx find="([nN])aj(maksimaln)ij([aeiou])" replaceWith="$2$3" />
+    <RegEx find="aj(optimaln)ij" replaceWith="$1" />
+    <RegEx find="aj(minimaln)ij" replaceWith="$1" />
+    <RegEx find="aj(maksimaln)ij" replaceWith="$1" />
```

Comment:
hmmmm. yes, you're right, as always. but furtherwise, upon testing I saw that **n** doesn't go away???

Najmaksimalniji becomes Nmaksimaln... it should have no N in the beginning of replaced word.
Do I have to keep it as it was, only squashed as your example?